pandas/tests/frame/test_misc_api.py

```diff
@@ -61,37 +61,32 @@ def test_join_index(self):
         # left / right
 
         f = self.frame.reindex(columns=['A', 'B'])[:10]
-        f2 = self.frame.reindex(columns=['C', 'D'])
+        f2 = self.frame.reindex(columns=['C', 'D'])[5:][::-1]
```

Comment:
you can easily indexing like this:

``self.frame.loc[self.frame.index[5::-1], ['C', 'D']]``